### DISCUSSION TITLE
Improvements to Creating and Displaying Attributes

### DISCUSSION THREAD

**Discussion by ayoubkhan558** [elementor/elementor#22171] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Currently, creating attributes in Elementor is a tedious and cumbersome process. The UI/UX is making it difficult. Also, It is almost impossible to create dynamic attributes from dynamic data or ACF. This is a pain point for users who want to create dynamic content and need to display it in a structured way using Elementor.

### Describe the solution you'd like

I would like to request a feature that allows users to easily create and manage attributes in Elementor easily. It should be possible to create attributes dynamically from ACF or other dynamic data sources, and they should be visible in the Elementor interface. This would greatly simplify the process of creating dynamic content with Elementor and make it more accessible to users.

ITt should be like this
![image](https://user-images.githubusercontent.com/99708427/233866404-3db6c040-432c-4da0-8044-b6daad6b4650.png)

### Describe alternatives you've considered

One alternative solution would be to use custom code to create attributes and dynamically populate them with data. However, this would require advanced coding skills and is not a viable option for many users. Another alternative is to use third-party plugins or widgets, but this can be expensive and may not be compatible with all themes or plugins.

### Additional context

Dynamic content is becoming increasingly important in web design, and Elementor should provide an easy and efficient way for users to create and manage it. By implementing this feature, Elementor can provide a more flexible and user-friendly platform for creating dynamic content, ultimately leading to a better user experience for website visitors.

**Comment by ghost** [comment]
+1

**Comment by Angelo-Verona** [comment]
I would like to expand on this request. Now these attributes are assigned to containers that's wrapping elements. We need a way how to add attributes to the item itself. for example iFrame. Cookie blockers needs to have targeting on certain elements to block them, but they cannot block them if they don't have custom attributes. We need to have a way how to add attribute to any element that its rendered, the best is the elements itself no just it's wrapper.
